packages/api/package.json

```diff
@@ -6,7 +6,7 @@
   },
   "scripts": {
     "generate": "pnpm run db:generate",
-    "db:generate": "dotenv -e .env.local -- prisma generate --data-proxy",
```

Comment:
Gördüğüm kadarıyla data-proxy kullanımı lokalde projeyi ayağa kaldırma sürecini karmaşıklaştırıyor. Serverless dünya kullanımı için ayrı bir komut eklenebilir diye düşünüyorum.

Reply:
Boyle tutmaya karar verirsek de readme dosyasinda gerekli adimlari aciklamak iyi olacaktir.